### DISCUSSION TITLE
How to redirect when accessing server domain/ip to certain page rather than the login page (self-host)

### DISCUSSION THREAD

**Discussion by MikeYates** [Budibase/budibase#11747] [Help]
Basically, I got budibase installed on a server (docker). When accessing my ip/domain eg: www.myapp.com, it will redirect to a login page for non logged-in / public user.

How do I manage that when accessing the domain, it redirects to a specific page/screen made inside budibase for public users. Then I could add a login button on that page/screen for users to log in.

**Comment by ConorWebb96** [comment]
Hey @MikeYates,

Is there a specific reason as to why you can't just direct them to the published app you want them to land at? (With the permissions set to public of course.)
<img width="1512" alt="Screenshot 2023-09-13 at 10 45 13" src="https://github.com/Budibase/budibase/assets/126772285/4304c5f1-4d05-4e04-860f-8bfd8b1f4882">

Can't you just send them directly to your public home screen? The below screenshot should hopefully describe what i mean a bit better. It shows both a non logged in and a logged in user being able to access the app.
<img width="1256" alt="Screenshot 2023-09-13 at 04 21 08" src="https://github.com/Budibase/budibase/assets/126772285/5de36430-01a5-40e3-8222-f5e0c621a23b">

If you have any permissions at all on the screen, this includes the tables permissions you either won't be able to access the screen and get redirected to the login page. Or a warning will appearing say not authenticated and can't view pieces of information on the page.

I hope this helps.

**Reply by MikeYates** [reply]
Ok yes, thats right. I wasn't thinking to redirect to the app url, I just thought we can only direct domain to the main ip of the server. 

Does this mean we can have multiple domain on a single server ip? for instance I have 2 apps published, each has its own domain. but how would the domain url of the login be like? since user database is shared across the apps, or should I host the other app on another server so that it has its own ip? Sorry I'm planning to publish my app soon, and got so much questions which I didnt thought about from the begining.

**Reply by ConorWebb96** [reply]
When you say domain do you just mean a slight change of the url path directing to your apps e.g. `/app-name/` or `/public-app/`. If you are talking about this yes, you can have as many apps as you want.

If you are talking about the initial segment `https://test....com` etc then I'm unsure. It may be locked to one of those per self-host instance so you may have to setup another host on a different server.

Sorry for being so vague on the second point its way outside my knowledge/general support knowledge.

I hope this helps.

**Reply by MikeYates** [reply]
You reply has always been helpful for me, and yes I do think the second point is correct thinking it logically. As the login URL path resides on the base server ip/builder/  `x.x.x.x:10000/builder/auth/login`

So the domain `appdomain.com` cannot be set on `x.x.x.x:1000/AppName/` As it starts at `/AppName/` directory, since the login directory is not under `/AppName/` ? Unless the `appdomain.com` is only a redirect and not the actual domain of the server ip.

I am super confused, this app building tool is very new to me.
